doc/user/pim.rst

```diff
@@ -343,10 +347,13 @@ MSDP can be setup in different ways:
 .. note::
 
    MSDP default peer is not implemented.
-   MSDP configuration is available under 'router pim'
 
+Commands available for MSDP
+---------------------------
+
+.. note::
 
-Commands available for MSDP:
+   MSDP configuration is available under 'router pim'
```

Comment:
Hate to say that: I know it's just moving back and forth, but since we are here, could you add a dot at the end? :smile: